### ISSUE TITLE
Composer package (3.1.32) contain file libs/libs - Verknüpfung.lnk

### ISSUE THREAD

**Issue by alebedev80** [smarty-php/smarty#449]
I've tried to deploy my project by rpm and rpm was broken because in Smarty library there is file with strange name `smarty/smarty/libs/libs - Verknüpfung.lnk`.

In you repo that file doesn't exist. Please rebuild the package.

**Comment by Psype** [comment]
Up for that stuff.
```
$ find . -iname "*.lnk"
./vendor/smarty/smarty/libs/libs - Verkn#U00fcpfung.lnk
```

**Comment by gdejong** [comment]
I just noticed this is fixed in `v3.1.33` (https://github.com/smarty-php/smarty/tree/v3.1.33)
